README.md

```diff
@@ -14,6 +14,15 @@ tweetnacl.h -> 02aee579efbe4f12ad8ffb23c6772841
 ## Fixes
 Change integer size in commit [b98e0ca](https://github.com/RIOT-OS/tweetnacl/commit/b98e0ca8beb9b802b5c0492d88846bd9de9946eb)
 
+Fix a warning on line 58 about comparison between two different integer
+sizes.
```

Comment:
Actually, if you want that degree of details, the change is on both. `int` is platform dependent size, and also unsigned, while `u32` is not signed and platform-independent size.